src/fixate/__main__.py

```diff
@@ -364,7 +364,9 @@ def run_main_program(test_script_path=None):
 
     args, unknown = parser.parse_known_args()
     if not args.disable_logs:
-        handler = RotateEachInstanceHandler("fixate.log", backupCount=10)
+        handler = RotateEachInstanceHandler(
+            fixate.config.CONFIG_DIRECTORY / "fixate.log", backupCount=10
+        )
         handler.setFormatter(
```

Comment:
`platformdirs.user_log_dir()`?
Generally good to keeps logs separate so you can easily cleanup/delete logs without risking deleting config. (I think...)

Reply:
Done